### ISSUE TITLE
run train1.py problem

### ISSUE THREAD

**Issue by JhengJheSie** [andabi/deep-voice-conversion#39]
![need more than 1 value to pick](https://user-images.githubusercontent.com/24805401/40303821-fc1585b4-5d26-11e8-9707-c761a691bfc0.PNG)
I change "queue = True" to "queue = False" in train1.py and *.wav to *.WAV in hparams.py,then i got this error.

**Comment by csubaiyang** [comment]
Yeah! I met the same problem as yours and I also make the *.wav replaced by *.WAV. have you solved it now? I'll appreciate that you can tell me how to solve it If you got some way.   Plz!!!

**Comment by JhengJheSie** [comment]
hi!
Maybe you can see
https://github.com/andabi/deep-voice-conversion/pulls
someone has fixed breaking code!

csubaiyang <notifications@github.com> 於 2019年1月10日 週四 上午10:50寫道：

**Comment by arjitkatare** [comment]
Replace : phn_file = wav_file.replace("WAV.wav", "PHN").replace("wav", "PHN")
to: phn_file = wav_file.replace("WAV.WAV", "PHN").replace("WAV", "PHN")
if your files have .WAV extension in the end.
